### PR TITLE
Added Config.json 

### PR DESCRIPTION
**Pull request by SeanHealy33**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -47,7 +54,8 @@ def getConversations():
 
 def closedRecently(conversation):
     t = int(time.time())
-    t = t - 86400
```
Comment (SeanHealy33) [inline main.py]: Magic numbers annoy me 😄

Comment (SeanHealy33) [inline main.py]: `timedelta(microseconds=-1).seconds` technically returns 86399 but it's close enough ¯\_(ツ)_/¯

Comment (looneym) [inline main.py]: Suggest changing this to 

``` python 
datetime.timedelta(hours=23.99).seconds
```

timedelta can't be used for exactly 24 hours as you pointed out but this makes it easy to adjust to 12 hours or whatever

Comment (looneym) [inline main.py]: Also change `day` variable name to something more generic. Maybe `offset` ?

**Pull request merged**